### PR TITLE
AwsParamStoreBootstrapConfigurationTest tests migrated to applicationContextRunner

### PR DESCRIPTION
**Pull request by MatejNedic**
#676 AwsParamStoreBootstrapConfigurationTest migrated.
Test for disabled added

### CODE REVIEW

File: spring-cloud-starter-aws-parameter-store-config/src/test/java/org/springframework/cloud/aws/autoconfigure/paramstore/AwsParamStoreBootstrapConfigurationTest.java [hunk 1]
```diff
@@ -35,42 +39,53 @@
...
+			.withConfiguration(
+					AutoConfigurations.of(AwsParamStoreBootstrapConfiguration.class));
 
 	@Test
 	void testWithStaticRegion() {
-		String region = "us-east-2";
-		AWSSimpleSystemsManagementClient awsSimpleClient = createParamStoreClient(region);
+		this.contextRunner.withPropertyValues("aws.paramstore.enabled:true",
+				"aws.paramstore.region:us-east-2").run(context -> {
+					assertThat(context.getBean(AwsParamStorePropertySourceLocator.class))
```
Comment (maciejwalkowiak) [inline spring-cloud-starter-aws-parameter-store-config/src/test/java/org/springframework/cloud/aws/autoconfigure/paramstore/AwsParamStoreBootstrapConfigurationTest.java:50]: This and all the other occurrences could be improved by using dedicated method to check if bean is there:
```
assertThat(context).hasSingleBean(AwsParamStorePropertySourceLocator.class);
```

**Review by maciejwalkowiak — CHANGES_REQUESTED** [review]
Thanks @MatejNedic for another nice contribution! 

I see a place for just a small improvement.

File: spring-cloud-starter-aws-parameter-store-config/src/test/java/org/springframework/cloud/aws/autoconfigure/paramstore/AwsParamStoreBootstrapConfigurationTest.java (hunk 1 above)
Comment (MatejNedic) [reply spring-cloud-starter-aws-parameter-store-config/src/test/java/org/springframework/cloud/aws/autoconfigure/paramstore/AwsParamStoreBootstrapConfigurationTest.java]: Something like this now?

Comment (maciejwalkowiak) [reply spring-cloud-starter-aws-parameter-store-config/src/test/java/org/springframework/cloud/aws/autoconfigure/paramstore/AwsParamStoreBootstrapConfigurationTest.java]: Yep, exactly!

**Review by maciejwalkowiak — APPROVED** [review]

**Comment by maciejwalkowiak** [comment]
Merged in 4705176f9f4fcec27d35fb49ea2475354167cec6

**Pull request closed without merging**